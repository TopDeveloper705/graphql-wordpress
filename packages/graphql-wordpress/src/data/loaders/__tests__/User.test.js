import getUserLoaders from 'data/loaders/User';

const User = getUserLoaders();

describe('Test User loader', () => {
  test('Load a user', async () => {
    const user = await User.load(1);
    expect(user.getID()).toMatchSnapshot();
    expect(user).toMatchSnapshot();
  });

  test('Load a user by slug', async () => {
    const user = await User.loadBySlug('admin');
    expect(user).toMatchSnapshot();
  });
});
