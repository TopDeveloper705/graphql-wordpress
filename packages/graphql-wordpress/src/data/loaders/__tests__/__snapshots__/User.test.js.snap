// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test User loader Load a user 1`] = `"VXNlcjox"`;

exports[`Test User loader Load a user 2`] = `
User {
  "id": 1,
}
`;

exports[`Test User loader Load a user by slug 1`] = `
User {
  "slug": "admin",
}
`;
